docs/datasets/diversity_reports/handwriting_ddr.md

```
@@ -0,0 +1,114 @@
+```bash
+# Run consensus review (models: gemini-2.5-pro, gemini-3-pro-preview,
+#   gpt-5.2, deepseek-r1-0528, grok-4)
+# Prompt: Review DDR for handwriting serving SigLIP-G4 (5 handwriting heads).
+# Output: Append to this file as Section 7.
+```
+
+**Consensus review not yet run.**
+
+---
+
+*Generated by `scripts/evaluate_dataset_diversity.py`*
```

markdownlint-cli2 (0.21.0)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)